Makefile

```diff
@@ -14,11 +14,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+.DEFAULT_GOAL := all
+
 .PHONY: all
 all: config.make
 	$(MAKE) all -C zdnn
 	$(MAKE) all -C tests
 
+help:
```

Comment:
Suggestion to add `.PHONY` to prevent any future conflict if a file called: `help` ever gets added. Additionally, `grep` options used are not available on zOS. Use of `egrep` and `cut` work on both zOS and LoZ. 

```
.PHONY: help
help:
	@echo "Available targets:"
	@egrep "^[a-z]+:" Makefile | cut -d':' -f1 | sort | xargs -n 1 echo "    "
```

Reply:
You have thought well around the corner!
Thank you!

Changed and worked.